zeebe/engine/src/test/java/io/camunda/zeebe/engine/util/TestStreams.java

```diff
@@ -95,10 +97,12 @@ public final class TestStreams {
   public TestStreams(
       final TemporaryFolder dataDirectory,
       final AutoCloseableRule closeables,
-      final ActorScheduler actorScheduler) {
+      final ActorScheduler actorScheduler,
+      final InstantSource clock) {
```

Comment:
Hm, this is going to cause conflicts with my PR :see_no_evil: Unfortunately I'm waiting for your review, so I guess you win :smile: